.github/workflows/e2e.yml

```diff
@@ -17,22 +17,26 @@ jobs:
     runs-on: ubuntu-20.04
 
     steps:
-      - name: Checkout subnet-evm repository
+      - name: Checkout awm-relayer repository
         uses: actions/checkout@v4
-        with:
-          repository: ava-labs/subnet-evm
-          ref: v0.5.8
-
+        
       - name: Set Go version
         run: |
           source ./scripts/versions.sh
-          GO_VERSION=$GO_VERSION >> $GITHUB_ENV
+          echo GO_VERSION=$GO_VERSION >> $GITHUB_ENV
+          echo SUBNET_EVM_VERSION=$SUBNET_EVM_VERSION >> $GITHUB_ENV
 
       - name: Setup Go
         uses: actions/setup-go@v5
         with:
           go-version: ${{ env.GO_VERSION }}
 
+      - name: Checkout subnet-evm repository
+        uses: actions/checkout@v4
+        with:
+          repository: ava-labs/subnet-evm
+          ref: v0.5.8 # TODO set with env.SUBNET_EVM_VERSION
```

Comment:
Is there a reason we can't do this TODO now?

Reply:
There's a breaking change, not exactly sure what it is, but created a ticket to investigate. https://github.com/ava-labs/awm-relayer/issues/109